index.html

```diff
@@ -250,7 +250,7 @@ <h1 class="skillHeading">Frameworks</h1>
         </div>
         <div class="skillslanguage">
           <div class="skillHeadingImg">
-            <img src="SVG/circle.svg" alt="" />
+            <img src="SVG/circle.svg" alt="icon" />
             <h1 class="skillHeading">Skills</h1>
```

Comment:
- [ ] Great job but I think you have the heading here is not follow the best practices for accesiblity so kindly update the h1 tag and put h2 or h3 instead of h1 you can just make a font weight or font size to it if you want to make it bigger but for semantic reasons and you should put one h1 tag per HTML page for SEO reasnons you can check this [link](https://ahrefs.com/blog/h1-tag/) for more information. 🚀💪🏻💯🔥☺

Reply:
thanks for the suggestion. I have updated the H1 to H2